### ISSUE TITLE
[Accessibility][DetailsList] No way to make DetailsList readOnly grid for screen reader

### ISSUE THREAD

**Issue by yaananth** [OfficeDev/office-ui-fabric-react#1258]
### Bug Report

- __Package version(s)__: latest

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Sure, might be late

Requested priority: Normal

#### Describe the issue:

DetailsList doesn't have a way to say it's readonly grid.
https://www.w3.org/TR/wai-aria/roles#grid

> A grid is considered editable unless otherwise specified. To make a grid read-only, set the aria-readonly attribute of the grid to true. 

https://www.w3.org/TR/wai-aria/states_and_properties#aria-readonly

**Ask regarding solution:**
How should we go about this? Just add a property that marks it readOnly? or allow native props to be applied to the grid so that consumers can adds other "data-*" or "aria-*" properties?
